presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -1435,8 +1470,16 @@ private static SqlTimestamp sqlTimestamp(long millisUtc)
     private static SqlTimestamp sqlTimestamp(String dateString)
             throws ParseException
     {
-        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
This change should be a separate commit.

Reply:
I don't think it's necessary because that function was added to test the `SEQUENCE` for `TIMESTAMP` return type support only. However, the tests are completely modified now to support `DATE` return type. Hence, I believe it should be put in the same commit, so people understand the context more clearly later on.

Reply:
@findepi, I have the same opinion as you at the beginning. But I realize that it wouldn't help in this case because every line that got touched in that pre commit will have to be touched again in the main commit anyways. I think it's fair as is.

@ngqtien, In general, we avoid having refactor and other change in the same file. Doing so makes reviewing this real changes much harder. But as I said in the previous paragraph, this is a special case.